I.write-tests/01-greet-people/greet-people.test.js

```diff
@@ -1,7 +1,8 @@
 test('print list of names prefixed with Hello', function () {
-    // Arrange
-
+   var greetPeople =require('./greet-people'); // Arrange
```

Comment:
Normally we put the line containing the `require` call at the top of the file, to separate it from our "normal" code.